src/sas/sascalc/fit/BumpsFitting.py

```diff
@@ -412,8 +415,14 @@ def abort_test():
     success = best is not None
     try:
         stderr = fitdriver.stderr() if success else None
-        cov = (fitdriver.cov() if not hasattr(fitdriver.fitter, 'state') else
-               np.cov(fitdriver.fitter.state.draw().points.T))
+        if not hasattr(fitdriver.fitter, 'state'):
```

Comment:
minor point... better to get rid of the not and swap the contents of the if/else